.circleci/config.yml

```diff
@@ -904,6 +926,7 @@ workflows:
       - test-libs-ballot-encoder
       - test-libs-ballot-interpreter-nh
       - test-libs-ballot-interpreter-vx
+      - test-libs-basics
```

Comment:
Make sure to have @carolinemodic add this to the required tests on Github after merging.